package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "steno",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
This change is unrelated to the security fix. We can manually apply the same.

Reply:
Ah, no. This is valid behavior. Somehow, this file is not correctly updated when v1.2.0 was released.